go.mod

```
@@ -5,26 +5,26 @@ go 1.18
 require (
 	github.com/diegoholiveira/jsonlogic/v3 v3.2.7
 	github.com/eclipse/paho.mqtt.golang v1.4.2
-	github.com/edgexfoundry/go-mod-bootstrap/v3 v3.0.0-dev.24
-	github.com/edgexfoundry/go-mod-core-contracts/v3 v3.0.0-dev.15
-	github.com/edgexfoundry/go-mod-messaging/v3 v3.0.0-dev.9
-	github.com/edgexfoundry/go-mod-registry/v3 v3.0.0-dev.3
+	github.com/edgexfoundry/go-mod-bootstrap/v3 v3.0.0-dev.35
```

Should be using the version these changes were introduced. Latest version introduce other changes that need accompanying code/config changes.
```suggestion
	github.com/edgexfoundry/go-mod-bootstrap/v3 v3.0.0-dev.29
```